from PyQt5 import QtCore

class ExceptionEvent(QtCore.QEvent):
    callback = None
